packages/app/src/overview/components/update-os-modals/update-available-modal/caution-section-container.component.tsx

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import React from "react"
+import styled from "styled-components"
+import { defineMessages } from "react-intl"
+import { FunctionComponent } from "App/__deprecated__/renderer/types/function-component.interface"
+import Text, {
+  TextDisplayStyle,
+} from "App/__deprecated__/renderer/components/core/text/text.component"
+
+const messages = defineMessages({
+  updateAvailableCautionSectionTitle: {
+    id: "module.overview.updateAvailableCautionSectionTitle",
+  },
+  updateAvailableCautionSectionSequenceUpdateDescription: {
+    id: "module.overview.updateAvailableCautionSectionSequenceUpdateDescription",
+  },
+  updateAvailableCautionSectionSingleUpdateDescription: {
+    id: "module.overview.updateAvailableCautionSectionSingleUpdateDescription",
+  },
+})
+
+interface CautionSectionProps {
+  isSingleRelease: boolean
+}
+
+const CautionSectionContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  margin-top: 2.4rem;
+
+  p {
+    text-align: left;
+  }
+`
+
+export const CautionSection: FunctionComponent<CautionSectionProps> = ({
```

Comment:
1. Maybe the file should be called just `caution-section-container.component.tsx`? In our codebase, name `container` indicates something different (a file that contains "connection" between component and redux values).
2. Similar to `CautionSectionContainer` - maybe `CautionSectionContainer` would be more appropriate?

Reply:
oh, this component should be `caution-section.component.tsx`. My mistake :) Thanks for review <3

Reply:
https://github.com/mudita/mudita-center/pull/1049/commits/b1e89e36ec965f086ebdbf036cf35ef7965c500d